src/video-player.js

```diff
@@ -201,7 +211,13 @@ class VideoPlayer extends Utils.mixin(Eventable) {
   }
 
   async _initLazyPlugins(options) {
+    this.loadingLazyPlugins = true;
+
     await this._initDash(options);
+
+    throttle(() => {
+      this.loadingLazyPlugins = false;
+    });
```

Comment:
I have 3 questions here:
- I believe we should be sending a timeout argument here, since the throttle function has no default timeout, right?
- I think this will never be executed eventually this will just create a function that does not ever get executed 
- I am curious why do you want to throttle this? it seems to me throttling it will actually be more expensive both memory and cpu over just executing it directly

Reply:
@tsi i might be completely wrong, but just based on code it is hard for me to understand how your function gets called, like is throttle calling the input function immediately? 

Anyway if you ask me based on your description it seems you are using throttle just as a way to setTimeout, both are indirect workarounds in my eyes, but i believe setTimeout is more explicit.

Reply:
@ehab-cl you are correct, both about the settimeout and about it being a workaround. I didn't like it but I needed a way to wait for the cycle to end for the error to never appear.
Anyway, pls see my latest version, IMO it makes much more sense.